src/components/Dashboard/AccountBanner.js

```diff
@@ -58,7 +58,7 @@ const getBannerAttributes = (
     title: 'Activate ANJ to be an active juror',
     paragraph: `You must activate at least ${formatUnits(minActiveBalance, {
       digits: decimals,
-    })}  ANJ to be drafted as a juror`,
+    })}  ANJ to be elegible for drafting`,
```

Comment:
@dizzypaty and I were discussing this; perhaps we could use "You must activate at least 10,000 ANJ to participate as a juror"?

Reply:
Yeah, that sounds good to me!